bundles/org.openhab.binding.mielecloud/src/main/java/org/openhab/binding/mielecloud/internal/config/servlet/CreateBridgeServlet.java

```diff
@@ -127,10 +127,11 @@ private Thing pairOrReconfigureBridge(String locale, ThingUID bridgeUid, String
                 .withProperty(Thing.PROPERTY_MODEL_ID, MIELE_CLOUD_BRIDGE_NAME)
                 .withProperty(MieleCloudBindingConstants.CONFIG_PARAM_LOCALE, locale)
                 .withProperty(MieleCloudBindingConstants.CONFIG_PARAM_EMAIL, email).build();
-        if (inbox.add(result)) {
-            return pairBridge(bridgeUid);
-        } else {
+        if (thingRegistry.get(bridgeUid) != null) {
```

Comment:
With this check in place, `reconfigureBridge` should be changed to accept the thing obtained here if it has been obtained.